test/geoloco/Geoloco_Regression.ipynb

```diff
@@ -0,0 +1,314 @@
+    "\n",
+    "mod08d3_non_production_info = {'collection' : Collection(id='C1257773477-LAADSCDUAT'),\n",
+    "                               'granule_id' : 'G1259320277-LAADSCDUAT',\n",
+    "                               'variable' : ['Aerosol_Optical_Depth_Land_Ocean_Mean'],\n",
+    "                               'downscale_size' : [2, 2]}\n",
+    "\n",
+    "geo_proj4_string = '+a=6378137.0 +b=6356752.3142451793 +no_defs +proj=latlong'\n",
+    "\n",
+    "resampling_string = 'NN'\n",
+    "\n",
+    "file_indicators = {'MOD021KM': 'EV_250_Aggr1km_RefSB_1.hdf',\n",
+    "                   'MOD35_L2': 'Cloud_Mask_1.hdf',\n",
+    "                   'MOD08_D3': 'Aerosol_Optical_Depth_Land_Ocean_Mean.hdf'}\n",
+    "\n",
+    "reference_data = {'MOD021KM': 'reference_data/MOD021KM.A2023001.0020.061.psrpcs_001701802061.EV_250_Aggr1km_RefSB_1.hdf',\n",
+    "                  'MOD35_L2': 'reference_data/MOD35_L2.A2023001.0020.061.psrpcs_001701881013.Cloud_Mask_1.hdf',\n",
+    "                  'MOD08_D3': 'reference_data/MOD08_D3.A2023001.061.psrpcs_001701881265.Aerosol_Optical_Depth_Land_Ocean_Mean.hdf'}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "722b554f-d69d-45bf-acfe-2ad255f875c0",
+   "metadata": {},
+   "source": [
+    "These selected collections and granules are only available in UAT environment. To minimize the output, all requests will utilize variable subsetting."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9864c2f7-2263-467f-8b33-4810e609cd41",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mod021km_geoloco_env = {Environment.LOCAL: mod021km_non_production_info,\n",
+    "                       Environment.UAT: mod021km_non_production_info,\n",
+    "                       Environment.SIT: mod021km_non_production_info}\n",
+    "mod35l2_geoloco_env = {Environment.LOCAL: mod35l2_non_production_info,\n",
+    "                       Environment.UAT: mod35l2_non_production_info,\n",
+    "                       Environment.SIT: mod35l2_non_production_info}\n",
+    "mod08d3_geoloco_env = {Environment.LOCAL: mod08d3_non_production_info,\n",
+    "                       Environment.UAT: mod08d3_non_production_info,\n",
+    "                       Environment.SIT: mod08d3_non_production_info}\n",
+    "\n",
+    "if harmony_environment in mod021km_geoloco_env:\n",
+    "    mod021km_geoloco_info =  mod021km_geoloco_env[harmony_environment]\n",
+    "else:\n",
+    "    mod021km_geoloco_info = None\n",
+    "\n",
+    "if harmony_environment in mod35l2_geoloco_env:\n",
+    "    mod35l2_geoloco_info =  mod35l2_geoloco_env[harmony_environment]\n",
+    "else:\n",
+    "    mod35l2_geoloco_info = None\n",
+    "\n",
+    "if harmony_environment in mod08d3_geoloco_env:\n",
+    "    mod08d3_geoloco_info =  mod08d3_geoloco_env[harmony_environment]\n",
+    "else:\n",
+    "    mod08d3_geoloco_info = None"
```

Comment:
One last nit (feel free to ignore):

You could simplify each of these `if/else` blocks using [the in-built dictionary get method](https://python-reference.readthedocs.io/en/latest/docs/dict/get.html#get):

```
mod021_geoloco_info = mod021mk_geoloco_env.get(harmony_environment)
# same for other collections
```

The `get` method has an implicit default value of `None`. You can also explicitly set it, if you wanted as a second argument: `value = dictionary.get('key', default_value)`